couchbase-entity/src/test/java/com/kaufland/CouchbaseBaseBinderProcessorKotlinTest.kt

```diff
@@ -12,74 +12,81 @@ class CouchbaseBaseBinderProcessorKotlinTest {
     fun testSucessSimpleMapper() {
         val compilation = compileKotlin(TestDataHelper.clazzAsJavaFileObjects("SimpleMapperTest"))
 
-        Assert.assertEquals(compilation.exitCode, KotlinCompilation.ExitCode.OK)
+        Assert.assertEquals(KotlinCompilation.ExitCode.OK, compilation.exitCode)
```

Comment:
asserEquals actually expects the "expected value" first and the real value after...